src/map/map.h

```diff
@@ -900,6 +900,8 @@ struct map_data {
 		unsigned pairship_startable : 1;
 		unsigned pairship_endable : 1;
 		uint32 noviewid; ///< noviewid (bitmask - @see enum equip_pos)
+		unsigned nostorage : 2;
+		unsigned nogstorage : 2;
```

Comment:
better put this fields before noviewid, for pack it into existing bitfields storage